From 44d38bad72f3fc1d4fbcf14abd5228d2a08c058b Mon Sep 17 00:00:00 2001
From: Alberto Ponces <ponces26@gmail.com>
Date: Fri, 28 Jan 2022 16:47:07 +0000
Subject: [PATCH 62/80] Remove Mi 11 Lite 5G (renoir) from the audio effects
 fix This should be replaced by the sound volume fix on the Qualcomm page of
 the TrebleApp

---
 rw-system.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/rw-system.sh b/rw-system.sh
index 216ebb5..6cf5f52 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -468,7 +468,6 @@ if getprop ro.vendor.build.fingerprint | grep -iq \
     -e motorola/hannah -e motorola/james -e motorola/pettyl -e xiaomi/cepheus \
     -e xiaomi/grus -e xiaomi/cereus -e xiaomi/cactus -e xiaomi/raphael -e xiaomi/davinci \
     -e xiaomi/ginkgo -e xiaomi/willow -e xiaomi/laurel_sprout -e xiaomi/andromeda \
-    -e iaomi/renoir \
     -e redmi/curtana -e -e redmi/joyeuse -e redmi/excalibur -e redmi/picasso \
     -e bq/Aquaris_M10 ; then
     mount -o bind /mnt/phh/empty_dir /vendor/lib64/soundfx
-- 
2.25.1

